caddyconfig/httpcaddyfile/httptype.go

```diff
@@ -477,10 +449,13 @@ func (ServerType) extractNamedRoutes(
 			continue
 		}
 
-		// zip up all the segments since ParseSegmentAsSubroute
-		// was designed to take a directive+
 		wholeSegment := caddyfile.Segment{}
 		for _, segment := range sb.block.Segments {
+			// Replace user-defined placeholder shorthands in extracted named routes
```

Comment:
Just a consistency thing

```suggestion
			// replace user-defined placeholder shorthands in extracted named routes
```